docs/admin/service-accounts-admin.md

```diff
@@ -60,9 +60,9 @@ It acts synchronously to modify pods as they are created or updated. When this p
 TokenController runs as part of controller-manager. It acts asynchronously. It:
 
 - observes serviceAccount creation and creates a corresponding Secret to allow API access.
```

Comment:
Rather than adding periods to the following lines, this period should be removed.